build.yaml

```diff
@@ -73,6 +73,37 @@ jobs:
             continueOnError: true
             displayName: publish e2e-failure-screenshots
 
+    # e2e run on linux - using docker.
+    - job: 'e2e_tests_linux'
+
+      pool:
+          vmImage: 'ubuntu-16.04'
+
+      steps:
+          - template: pipeline/e2e-test-linux-docker.yaml
+
+    - job: 'build_electron_app'
```

Comment:
The separate build_electron_app step was reverted in master since you started this PR, so I think for this PR's purposes it probably makes sense to omit the separate job for it